content/help/faq/distributed-tracing/mixer-role.md

```diff
@@ -0,0 +1,10 @@
+---
+title: What role does Mixer play in the Istio tracing story?
+weight: 90
+---
+
+By default, Mixer participates in tracing by generating its own spans for requests that are already selected for tracing by Envoy proxies. This enables operators to observe the participation of the mixer-based policy enforcement mechanisms within the mesh. If the `istio-policy` configuration is disabled mesh-wide, Mixer does not participate in tracing in this way.
+
+Mixer can also be used to generate trace spans for data plane traffic. Mixer’s Stackdriver adapter is the first adapter to support this capability. Soon, we will support Zipkin (and Zipkin API compatible backends) in a similar fashion.
```

Comment:
I like how you talk about istio-policy in earlier paragraph vs mixer... to user, it is confusing which mixer component are you referring to (either policy or telemetry or both).  I would like to see the clarity in this paragraph and the one below it. :)